<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
    <meta charset="UTF-8"/>
    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="access" content="Yes">

    

    <meta name="twitter:card" content="summary"/>

    <meta name="twitter:title" content="Immersive structural biology: a new approach to hybrid modeling of mac"/>

    <meta name="twitter:site" content="@SpringerLink"/>

    <meta name="twitter:description" content="Advanced biophysical imaging techniques, such as cryo-electron microscopy or tomography, enable 3D volumetric reconstructions of large macromolecular complexes in a near-native environment...."/>

    <meta name="twitter:image" content="https://static-content.springer.com/cover/journal/10055/13/4.jpg"/>

    <meta name="twitter:image:alt" content="Content cover image"/>

    <meta name="journal_id" content="10055"/>

    <meta name="dc.title" content="Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies"/>

    <meta name="dc.source" content="Virtual Reality 2009 13:4"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="Springer"/>

    <meta name="dc.date" content="2009-08-26"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2009 The Author(s)"/>

    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>

    <meta name="dc.description" content="Advanced biophysical imaging techniques, such as cryo-electron microscopy or tomography, enable 3D volumetric reconstructions of large macromolecular complexes in a near-native environment. However, pure volumetric data is insufficient for a detailed understanding of the underlying protein&#8211;protein interactions. This obstacle can be overcome by assembling an atomic model of the whole protein complex from known atomic structures, which are available from either X-ray crystallography or homology modeling. Due to many factors such as noise, conformational variability, experimental artifacts, and inexact model structures, existing automatic docking procedures are known to report false positives for a significant number of cases. The present paper focuses on a new technique to combine an offline exhaustive search algorithm with interactive visualization, collision detection, and haptic rendering. The resulting software system is highly immersive and allows the user to efficiently solve even difficult multi-resolution docking problems. Stereoscopic viewing, combined with head tracking and force feedback, generates an ideal virtual environment for true interaction with and solution of hybrid biomolecular modeling problems."/>

    <meta name="prism.issn" content="1434-9957"/>

    <meta name="prism.publicationName" content="Virtual Reality"/>

    <meta name="prism.publicationDate" content="2009-08-26"/>

    <meta name="prism.volume" content="13"/>

    <meta name="prism.number" content="4"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="245"/>

    <meta name="prism.endingPage" content="255"/>

    <meta name="prism.copyright" content="2009 The Author(s)"/>

    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>

    <meta name="prism.url" content="https://link.springer.com/article/10.1007/s10055-009-0129-y"/>

    <meta name="prism.doi" content="doi:10.1007/s10055-009-0129-y"/>

    <meta name="citation_pdf_url" content="https://link.springer.com/content/pdf/10.1007/s10055-009-0129-y.pdf"/>

    <meta name="citation_fulltext_html_url" content="https://link.springer.com/article/10.1007/s10055-009-0129-y"/>

    <meta name="citation_journal_title" content="Virtual Reality"/>

    <meta name="citation_journal_abbrev" content="Virtual Reality"/>

    <meta name="citation_publisher" content="Springer-Verlag"/>

    <meta name="citation_issn" content="1434-9957"/>

    <meta name="citation_title" content="Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies"/>

    <meta name="citation_volume" content="13"/>

    <meta name="citation_issue" content="4"/>

    <meta name="citation_publication_date" content="2009/12"/>

    <meta name="citation_online_date" content="2009/08/26"/>

    <meta name="citation_firstpage" content="245"/>

    <meta name="citation_lastpage" content="255"/>

    <meta name="citation_article_type" content="Original Article"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="citation_language" content="en"/>

    <meta name="dc.identifier" content="doi:10.1007/s10055-009-0129-y"/>

    <meta name="DOI" content="10.1007/s10055-009-0129-y"/>

    <meta name="citation_doi" content="10.1007/s10055-009-0129-y"/>

    <meta name="description" content="Advanced biophysical imaging techniques, such as cryo-electron microscopy or tomography, enable 3D volumetric reconstructions of large macromolecular compl"/>

    <meta name="dc.creator" content="Jochen Heyd"/>

    <meta name="dc.creator" content="Stefan Birmanns"/>

    <meta name="dc.subject" content="Computer Graphics"/>

    <meta name="dc.subject" content="Artificial Intelligence"/>

    <meta name="dc.subject" content="Artificial Intelligence"/>

    <meta name="dc.subject" content="Image Processing and Computer Vision"/>

    <meta name="dc.subject" content="User Interfaces and Human Computer Interaction"/>

    <meta name="citation_reference" content="citation_journal_title=ACM Trans Math Software; citation_title=The quickhull algorithm for convex hulls; citation_author=CB Barber, DP Dobkin, HT Huhdanpaa; citation_volume=22; citation_publication_date=1996; citation_pages=469-483; citation_doi=10.1145/235815.235821; citation_id=CR25"/>

    <meta name="citation_reference" content="Bayazit OB, Song G, Amato NM (2001) Ligand binding with OBPRM and haptic user input. In: 2001 IEEE international conference on robotics and automation. Seoul, Korea"/>

    <meta name="citation_reference" content="citation_journal_title=J Struct Biol; citation_title=Interactive fitting augmented by force-feedback and virtual reality; citation_author=S Birmanns, W Wriggers; citation_volume=144; citation_publication_date=2003; citation_pages=123-131; citation_doi=10.1016/j.jsb.2003.09.018; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=J Struct Biol; citation_title=Multi-resolution anchor-point registration of biomolecular assemblies and their components; citation_author=S Birmanns, W Wriggers; citation_volume=157; citation_publication_date=2007; citation_pages=271-280; citation_doi=10.1016/j.jsb.2006.08.008; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Fast fitting of atomic structures to low-resolution electron density maps by surface overlap maximization; citation_author=H Ceulemans, RB Russell; citation_volume=338; citation_publication_date=2004; citation_pages=783-793; citation_doi=10.1016/j.jmb.2004.02.066; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Multi-resolution contour-based fitting of macromolecular structures; citation_author=P Chac&#243;n, W Wriggers; citation_volume=317; citation_publication_date=2002; citation_pages=375-384; citation_doi=10.1006/jmbi.2002.5438; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Biophys Biomol Struct; citation_title=Single-particle imaging of macromolecules by cryo-electron microscopy; citation_author=J Frank; citation_volume=31; citation_publication_date=2002; citation_pages=303-319; citation_doi=10.1146/annurev.biophys.31.082901.134202; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_title=Three-dimensional electron microscopy of macromolecular assemblies; citation_publication_date=2006; citation_id=CR7; citation_author=J Frank; citation_publisher=Oxford University Press"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ADP_EM: fast exhaustive multi-resolution docking for high-throughput coverage; citation_author=JI Garz&#243;n, JA Kovacs, R Abagyan, P Chac&#243;n; citation_volume=23; citation_publication_date=2007; citation_pages=427-433; citation_doi=10.1093/bioinformatics/btl625; citation_id=CR10"/>

    <meta name="citation_reference" content="Intel Corporation (1994&#8211;2008) Intel math kernel library 10.0"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Bridging the information gap: computational tools for intermediate resolution structure interpretation; citation_author=W Jiang, ML Baker, SJ Ludtke, W Chiu; citation_volume=308; citation_publication_date=2001; citation_pages=1033-1044; citation_doi=10.1006/jmbi.2001.4633; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Proteins; citation_title=Backbonebackbone geometry of tertiary contacts between &#945;-helices; citation_author=P K&#228;llblad, PM Dean; citation_volume=56; citation_publication_date=2004; citation_pages=693-703; citation_doi=10.1002/prot.20201; citation_id=CR27"/>

    <meta name="citation_reference" content="Kleywegt GJ, Zou JY, Kjeldgaard M, Jones TA (2001) Around O. International tables for crystallography, vol F, chap 17.1. Kluwer, Dordrecht, pp 353&#8211;356, 366&#8211;367"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=ORC and Cdc6p interact and determine the frequency of initiation of DNA replication in the genome; citation_author=C Liang, M Weinreich, B Stillman; citation_volume=81; citation_publication_date=1995; citation_pages=667-676; citation_doi=10.1016/0092-8674(95)90528-6; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Structure and function of Cdc6/Cdc18: implications for origin recognition and checkpoint control; citation_author=J Liu, CL Smith, D DeRyckere, K DeAngelis, GS Martin, JM Berger; citation_volume=6; citation_publication_date=2000; citation_pages=637-648; citation_doi=10.1016/S1097-2765(00)00062-9; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Biochem; citation_title=Structural studies by electron tomography: from cells to molecules; citation_author=V Lu&#269;i&#263;, F F&#246;rster, W Baumeister; citation_volume=74; citation_publication_date=2005; citation_pages=833-865; citation_doi=10.1146/annurev.biochem.73.011303.074112; citation_id=CR2"/>

    <meta name="citation_reference" content="Novint Technologies Inc. (2008) The Falcon haptic device. 
                    http://www.novint.com
                    
                  
                        "/>

    <meta name="citation_reference" content="Ouh-young M, Pique M, Hughes J, Srinivasan N, Brooks FP Jr (1988) Using a manipulator for force display in molecular docking. In: 1988 IEEE robotics and automation conference, vol 3. Philadelphia, PA, pp 1824&#8211;1829"/>

    <meta name="citation_reference" content="citation_journal_title=Curr Opin Struct Biol; citation_title=Analysis of X-ray and neutron scattering from biomacromolecular solutions; citation_author=MV Petoukhov, DI Svergun; citation_volume=17; citation_publication_date=2007; citation_pages=562-571; citation_doi=10.1016/j.sbi.2007.06.009; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr D; citation_title=Docking structures of domains into maps from cryo-electron microscopy using local correlation; citation_author=AM Roseman; citation_volume=56; citation_publication_date=2000; citation_pages=1332-1340; citation_doi=10.1107/S0907444900010908; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr D; citation_title=Fitting atomic models into electron-microscopy maps; citation_author=MG Rossmann; citation_volume=56; citation_publication_date=2000; citation_pages=1341-1349; citation_doi=10.1107/S0907444900009562; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Structure; citation_title=Combining X-ray crystallography and electron microscopy; citation_author=MG Rossmann, MC Morais, PG Leiman, W Zhang; citation_volume=13; citation_publication_date=2005; citation_pages=355-362; citation_doi=10.1016/j.str.2005.01.005; citation_id=CR6"/>

    <meta name="citation_reference" content="Short J (2006) MRC Cambridge Image Processing System 28 Sep 2006"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Struct Mol Biol; citation_title=ATPase-dependent cooperative binding of ORC and Cdc6 to origin DNA; citation_author=C Speck, Z Chen, H Li, B Stillman; citation_volume=12; citation_publication_date=2005; citation_pages=965-971; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=J Struct Biol; citation_title=The future is hybrid; citation_author=AC Steven, W Baumeister; citation_volume=163; citation_publication_date=2008; citation_pages=186-195; citation_doi=10.1016/j.jsb.2008.06.002; citation_id=CR4"/>

    <meta name="citation_reference" content="The worldwide Protein Data Bank (2008) Protein data bank contents guide: atomic coordinate entry format description"/>

    <meta name="citation_reference" content="citation_journal_title=J Struct Biol; citation_title=Quantitative fitting of atomic models into observed densities by electron microscopy; citation_author=N Volkmann, D Hanein; citation_volume=125; citation_publication_date=1999; citation_pages=176-184; citation_doi=10.1006/jsbi.1998.4074; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Structure; citation_title=Modeling tricks and fitting techniques for multiresolution structures; citation_author=W Wriggers, P Chac&#243;n; citation_volume=9; citation_publication_date=2001; citation_pages=779-788; citation_doi=10.1016/S0969-2126(01)00648-7; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=J Struct Biol; citation_title=Situs: a package for docking crystal structures into low-resolution maps from electron microscopy; citation_author=W Wriggers, RA Milligan, JA McCammon; citation_volume=125; citation_publication_date=1999; citation_pages=185-195; citation_doi=10.1006/jsbi.1998.4080; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=J Struct Biol; citation_title=A core-weighted fitting method for docking atomic structures into low-resolution maps: application to cryo-electron microscopy; citation_author=X Wu, JLS Milne, MJ Borgnia, AV Rostapshov, S Subramaniam, BR Brooks; citation_volume=141; citation_publication_date=2003; citation_pages=63-76; citation_doi=10.1016/S1047-8477(02)00570-1; citation_id=CR15"/>

    <meta name="citation_author" content="Jochen Heyd"/>

    <meta name="citation_author_email" content="jheyd@biomachina.org"/>

    <meta name="citation_author_institution" content="School of Health Information Sciences, University of Texas at Houston, Houston, USA"/>

    <meta name="citation_author" content="Stefan Birmanns"/>

    <meta name="citation_author_email" content="sbirmanns@biomachina.org"/>

    <meta name="citation_author_institution" content="School of Health Information Sciences, University of Texas at Houston, Houston, USA"/>

    <meta name="citation_springer_api_url" content="http://api.springer.com/metadata/pam?q=doi:10.1007/s10055-009-0129-y&amp;api_key="/>

    <meta name="format-detection" content="telephone=no"/>

    <meta name="citation_cover_date" content="2009/12/01"/>


    
        <meta property="og:url" content="https://link.springer.com/article/10.1007/s10055-009-0129-y"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Virtual Reality"/>
        <meta property="og:title" content="Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies"/>
        <meta property="og:description" content="Advanced biophysical imaging techniques, such as cryo-electron microscopy or tomography, enable 3D volumetric reconstructions of large macromolecular complexes in a near-native environment. However, pure volumetric data is insufficient for a detailed understanding of the underlying protein–protein interactions. This obstacle can be overcome by assembling an atomic model of the whole protein complex from known atomic structures, which are available from either X-ray crystallography or homology modeling. Due to many factors such as noise, conformational variability, experimental artifacts, and inexact model structures, existing automatic docking procedures are known to report false positives for a significant number of cases. The present paper focuses on a new technique to combine an offline exhaustive search algorithm with interactive visualization, collision detection, and haptic rendering. The resulting software system is highly immersive and allows the user to efficiently solve even difficult multi-resolution docking problems. Stereoscopic viewing, combined with head tracking and force feedback, generates an ideal virtual environment for true interaction with and solution of hybrid biomolecular modeling problems."/>
        <meta property="og:image" content="https://media.springernature.com/w110/springer-static/cover/journal/10055.jpg"/>
    

    <title>Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies | SpringerLink</title>

    <link rel="shortcut icon" href=/oscar-static/images/favicons/springerlink/favicon-eb9f5576a3.ico />
<link rel="icon" sizes="16x16 32x32 48x48" href=/oscar-static/images/favicons/springerlink/favicon-eb9f5576a3.ico />
<link rel="icon" sizes="16x16" type="image/png" href=/oscar-static/images/favicons/springerlink/favicon-16x16-8bd8c1c945.png />
<link rel="icon" sizes="32x32" type="image/png" href=/oscar-static/images/favicons/springerlink/favicon-32x32-61a52d80ab.png />
<link rel="icon" sizes="48x48" type="image/png" href=/oscar-static/images/favicons/springerlink/favicon-48x48-0ec46b6b10.png />
<link rel="apple-touch-icon" href=/oscar-static/images/favicons/springerlink/app-icon-iphone@3x-f259d46347.png />
<link rel="apple-touch-icon" sizes="72x72" href=/oscar-static/images/favicons/springerlink/ic_launcher_hdpi-f77cda7f65.png />
<link rel="apple-touch-icon" sizes="76x76" href=/oscar-static/images/favicons/springerlink/app-icon-ipad-c3fd26520d.png />
<link rel="apple-touch-icon" sizes="114x114" href=/oscar-static/images/favicons/springerlink/app-icon-114x114-3d7d4cf9f3.png />
<link rel="apple-touch-icon" sizes="120x120" href=/oscar-static/images/favicons/springerlink/app-icon-iphone@2x-67b35150b3.png />
<link rel="apple-touch-icon" sizes="144x144" href=/oscar-static/images/favicons/springerlink/ic_launcher_xxhdpi-986442de7b.png />
<link rel="apple-touch-icon" sizes="152x152" href=/oscar-static/images/favicons/springerlink/app-icon-ipad@2x-677ba24d04.png />
<link rel="apple-touch-icon" sizes="180x180" href=/oscar-static/images/favicons/springerlink/app-icon-iphone@3x-f259d46347.png />


    
    <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>

    <link rel="stylesheet" href=/oscar-static/app-springerlink/css/core-article-b0cd12fb00.css media="screen">
    <link rel="stylesheet" id="js-mustard" href=/oscar-static/app-springerlink/css/enhanced-article-6aad73fdd0.css media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
    

    
    <script type="text/javascript">
        window.dataLayer = [{"Country":"DK","doi":"10.1007-s10055-009-0129-y","Journal Title":"Virtual Reality","Journal Id":10055,"Keywords":"Structural biology, Docking, Multi-resolution modeling, Electron microscopy, Haptic rendering, Virtual reality","kwrd":["Structural_biology","Docking","Multi-resolution_modeling","Electron_microscopy","Haptic_rendering","Virtual_reality"],"Labs":"Y","ksg":"Krux.segments","kuid":"Krux.uid","Has Body":"Y","Features":["doNotAutoAssociate"],"Open Access":"Y","hasAccess":"Y","bypassPaywall":"N","user":{"license":{"businessPartnerID":["1600006921","2000421697","3000092219","3000209025","3000236495"],"businessPartnerIDString":"1600006921|2000421697|3000092219|3000209025|3000236495"}},"Access Type":"subscription","Bpids":"1600006921, 2000421697, 3000092219, 3000209025, 3000236495","Bpnames":"Copenhagen University Library Royal Danish Library Copenhagen, DEFF Consortium Danish National Library Authority, Det Kongelige Bibliotek The Royal Library, DEFF Danish Agency for Culture, 1236 DEFF LNCS","BPID":["1600006921","2000421697","3000092219","3000209025","3000236495"],"VG Wort Identifier":"vgzm.415900-10.1007-s10055-009-0129-y","Full HTML":"Y","Subject Codes":["SCI","SCI22013","SCI21000","SCI22021","SCI18067"],"pmc":["I","I22013","I21000","I22021","I18067"],"session":{"authentication":{"loginStatus":"N"},"attributes":{"edition":"academic"}},"content":{"serial":{"eissn":"1434-9957","pissn":"1359-4338"},"type":"Article","category":{"pmc":{"primarySubject":"Computer Science","primarySubjectCode":"I","secondarySubjects":{"1":"Computer Graphics","2":"Artificial Intelligence","3":"Artificial Intelligence","4":"Image Processing and Computer Vision","5":"User Interfaces and Human Computer Interaction"},"secondarySubjectCodes":{"1":"I22013","2":"I21000","3":"I21000","4":"I22021","5":"I18067"}},"sucode":"SC6"},"attributes":{"deliveryPlatform":"oscar"}},"Event Category":"Article","GA Key":"UA-26408784-1","DOI":"10.1007/s10055-009-0129-y","Page":"article","page":{"attributes":{"environment":"live"}}}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script>


    


<script src="/oscar-static/js/jquery-220afd743d.js" id="jquery"></script>

<script>
    function OptanonWrapper() {
        dataLayer.push({
            'event' : 'onetrustActive'
        });
    }
</script>


    <script data-test="onetrust-link" type="text/javascript" src="https://cdn.cookielaw.org/consent/6b2ec9cd-5ace-4387-96d2-963e596401c6.js" charset="UTF-8"></script>





    
    
        <!-- Google Tag Manager -->
        <script data-test="gtm-head">
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-WCF9Z9');
        </script>
        <!-- End Google Tag Manager -->
    


    <script class="js-entry">
    (function(w, d) {
        window.config = window.config || {};
        window.config.mustardcut = false;

        var ctmLinkEl = d.getElementById('js-mustard');

        if (ctmLinkEl && w.matchMedia && w.matchMedia(ctmLinkEl.media).matches) {
            window.config.mustardcut = true;

            
            
            
                window.Component = {};
            

            var currentScript = d.currentScript || d.head.querySelector('script.js-entry');

            
            function catchNoModuleSupport() {
                var scriptEl = d.createElement('script');
                return (!('noModule' in scriptEl) && 'onbeforeload' in scriptEl)
            }

            var headScripts = [
                { 'src' : '/oscar-static/js/polyfill-es5-bundle-ab77bcf8ba.js', 'async': false, 'module': false},
                { 'src' : '/oscar-static/js/airbrake-es5-bundle-6b407673fa.js', 'async': false, 'module': false},
                { 'src' : '/oscar-static/js/airbrake-es6-bundle-e7bd4589ff.js', 'async': false, 'module': true}
            ];

            var bodyScripts = [
                { 'src' : '/oscar-static/js/app-es5-bundle-867d07d044.js', 'async': false, 'module': false},
                { 'src' : '/oscar-static/js/app-es6-bundle-8ebcb7376d.js', 'async': false, 'module': true}
                
                
                    , { 'src' : '/oscar-static/js/global-article-es5-bundle-12442a199c.js', 'async': false, 'module': false},
                    { 'src' : '/oscar-static/js/global-article-es6-bundle-7ae1776912.js', 'async': false, 'module': true}
                
            ];

            function createScript(script) {
                var scriptEl = d.createElement('script');
                scriptEl.src = script.src;
                scriptEl.async = script.async;
                if (script.module === true) {
                    scriptEl.type = "module";
                    if (catchNoModuleSupport()) {
                        scriptEl.src = '';
                    }
                } else if (script.module === false) {
                    scriptEl.setAttribute('nomodule', true)
                }
                if (script.charset) {
                    scriptEl.setAttribute('charset', script.charset);
                }

                return scriptEl;
            }

            for (var i=0; i<headScripts.length; ++i) {
                var scriptEl = createScript(headScripts[i]);
                currentScript.parentNode.insertBefore(scriptEl, currentScript.nextSibling);
            }

            d.addEventListener('DOMContentLoaded', function() {
                for (var i=0; i<bodyScripts.length; ++i) {
                    var scriptEl = createScript(bodyScripts[i]);
                    d.body.appendChild(scriptEl);
                }
            });

            // Webfont repeat view
            var config = w.config;
            if (config && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                d.documentElement.className += ' webfonts-loaded';
            }
        }
    })(window, document);
</script>

</head>
<body class="shared-article-renderer">
    
    
    
        <!-- Google Tag Manager (noscript) -->
        <noscript data-test="gtm-body">
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WCF9Z9"
            height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <!-- End Google Tag Manager (noscript) -->
    


    <div class="u-vh-full">
        
    <aside class="c-ad c-ad--728x90" data-test="springer-doubleclick-ad">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/springerlink/10055/article" data-gpt-sizes="728x90" data-gpt-targeting="pos=LB1;articleid=129;"></div>
        </div>
    </aside>

<div class="u-position-relative">
    <header class="c-header u-mb-24" data-test="publisher-header">
        <div class="c-header__container">
            <div class="c-header__brand">
                
    <a id="logo" class="u-display-block" href="/" title="Go to homepage" data-test="springerlink-logo">
        <picture>
            <source type="image/svg+xml" srcset=/oscar-static/images/springerlink/svg/springerlink-253e23a83d.svg>
            <img src=/oscar-static/images/springerlink/png/springerlink-1db8a5b8b1.png alt="SpringerLink" width="148" height="30" data-test="header-academic">
        </picture>
        
        
    </a>


            </div>
            <div class="c-header__navigation">
                
    
        <button type="button"
                class="c-header__link u-button-reset u-mr-24"
                data-expander
                data-expander-target="#popup-search"
                data-expander-autofocus="firstTabbable"
                data-test="header-search-button">
            <span class="u-display-flex u-align-items-center">
                Search
                <svg class="c-icon u-ml-8" width="22" height="22" aria-hidden="true" focusable="false">
                    <use xlink:href="#global-icon-search"></use>
                </svg>
            </span>
        </button>
        <nav>
            <ul class="c-header__menu">
                
                <li class="c-header__item">
                    <a data-test="login-link" class="c-header__link" href="//link.springer.com/signup-login?previousUrl=https%3A%2F%2Flink.springer.com%2Farticle%2F10.1007%2Fs10055-009-0129-y">Log in</a>
                </li>
                

                
            </ul>
        </nav>
    

    



            </div>
        </div>
    </header>

    
        <div id="popup-search" class="c-popup-search u-mb-16 js-header-search u-js-hide">
            <div class="c-popup-search__content">
                <div class="u-container">
                    <div class="c-popup-search__container" data-test="springerlink-popup-search">
                        <div class="app-search">
    <form role="search" method="GET" action="/search" >
        <label for="search" class="app-search__label">Search SpringerLink</label>
        <div class="app-search__content">
            <input id="search" class="app-search__input" data-search-input autocomplete="off" role="textbox" name="query" type="text" value="">
            <button class="app-search__button" type="submit">
                <span class="u-visually-hidden">Search</span>
                <svg class="c-icon" width="14" height="14" aria-hidden="true" focusable="false">
                    <use xlink:href="#global-icon-search"></use>
                </svg>
            </button>
            
                <input type="hidden" name="searchType" value="publisherSearch">
            
            
        </div>
    </form>
</div>

                    </div>
                </div>
            </div>
        </div>
    
</div>

        

    <div class="u-container u-mt-32 u-mb-32 u-clearfix" id="main-content" data-component="article-container">
        <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
            
                
                <div class="c-context-bar u-hide" data-component="context-bar" aria-hidden="true">
                    <div class="c-context-bar__container u-container">
                        <div class="c-context-bar__title">
                            Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies
                        </div>
                        
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://link.springer.com/content/pdf/10.1007/s10055-009-0129-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="button">
            
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            
        </a>
    </div>
    

                    </div>
                </div>
            
            
            
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://link.springer.com/content/pdf/10.1007/s10055-009-0129-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="button">
            
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            
        </a>
    </div>
    

            <article itemscope itemtype="http://schema.org/ScholarlyArticle" lang="en">
                <div class="c-article-header">
                    <header>
                        <ul class="c-article-identifiers" data-test="article-identifier">
                            
    
        <li class="c-article-identifiers__item" data-test="article-category">Original Article</li>
    
    
        <li class="c-article-identifiers__item">
            <span class="c-article-identifiers__open" data-test="open-access">Open Access</span>
        </li>
    
    

                            <li class="c-article-identifiers__item"><a href="#article-info" data-track="click" data-track-action="publication date" data-track-label="link">Published: <time datetime="2009-08-26" itemprop="datePublished">26 August 2009</time></a></li>
                        </ul>

                        
                        <h1 class="c-article-title" data-test="article-title" data-article-title="" itemprop="name headline">Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies</h1>
                        <ul class="c-author-list js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jochen-Heyd" data-author-popup="auth-Jochen-Heyd">Jochen Heyd</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="University of Texas at Houston" /><meta itemprop="address" content="grid.267308.8, 0000000092062401, School of Health Information Sciences, University of Texas at Houston, 7000 Fannin St., Houston, TX, 77030, USA" /></span></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stefan-Birmanns" data-author-popup="auth-Stefan-Birmanns" data-corresp-id="c1">Stefan Birmanns<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-email"></use></svg></a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="University of Texas at Houston" /><meta itemprop="address" content="grid.267308.8, 0000000092062401, School of Health Information Sciences, University of Texas at Houston, 7000 Fannin St., Houston, TX, 77030, USA" /></span></sup> </li></ul>
                        <p class="c-article-info-details" data-container-section="info">
                            
    <a data-test="journal-link" href="/journal/10055"><i data-test="journal-title">Virtual Reality</i></a>

                            <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 13</b>, <span class="u-visually-hidden">pages</span><span itemprop="pageStart">245</span>–<span itemprop="pageEnd">255</span>(<span data-test="article-publication-year">2009</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                        </p>
                        
    

                        <div data-test="article-metrics">
                            <div id="altmetric-container">
    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">945 <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">11 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/article/10.1007%2Fs10055-009-0129-y/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
</div>

                        </div>
                        
                        
                        
                    </header>
                </div>

                <div data-article-body="true" data-track-component="article body" class="c-article-body">
                    <section aria-labelledby="Abs1" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Advanced biophysical imaging techniques, such as cryo-electron microscopy or tomography, enable 3D volumetric reconstructions of large macromolecular complexes in a near-native environment. However, pure volumetric data is insufficient for a detailed understanding of the underlying protein–protein interactions. This obstacle can be overcome by assembling an atomic model of the whole protein complex from known atomic structures, which are available from either X-ray crystallography or homology modeling. Due to many factors such as noise, conformational variability, experimental artifacts, and inexact model structures, existing automatic docking procedures are known to report false positives for a significant number of cases. The present paper focuses on a new technique to combine an offline exhaustive search algorithm with interactive visualization, collision detection, and haptic rendering. The resulting software system is highly immersive and allows the user to efficiently solve even difficult multi-resolution docking problems. Stereoscopic viewing, combined with head tracking and force feedback, generates an ideal virtual environment for true interaction with and solution of hybrid biomolecular modeling problems.</p></div></div></section>
                    
    


                    

                    

                    
                        
                            <section aria-labelledby="Sec1"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>A key to understand the function of biological systems is the visualization of their natural state, ideally in a natural environment. At a molecular level, this is challenging. Traditional experimental techniques, such as X-ray crystallography, can provide the atomic structure of proteins, but only by removing them from their native surroundings and forcing them into a crystal lattice. Over the past decade, microscopy techniques have emerged as alternatives to these traditional structure determination methods, with the advantage of visualizing large multi-component molecules in a near-native state. Given the current focus of structural biology on interactions between proteins and better understanding of large protein complexes, cryo-electron microscopy (cryo-EM) has become a valuable tool (Frank <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2002" title="Frank J (2002) Single-particle imaging of macromolecules by cryo-electron microscopy. Annu Rev Biophys Biomol Struct 31:303–319" href="/article/10.1007/s10055-009-0129-y#ref-CR1" id="ref-link-section-d20606e323">2002</a>). Both image acquisition techniques and the computational synthesis of 3D volumetric models from micrographs have advanced considerably. 3D reconstructions of large protein complexes or even individual proteins can now be obtained (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig1">1</a>). While cryo-EM, thus, offers numerous advantages (small sample size, no need to crystallize, no packing effects, etc.), its main drawback is its inability to attain atomic resolution. Related techniques, such as cryo-electron tomography (cryo-ET; Lučić et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2005" title="Lučić V, Förster F, Baumeister W (2005) Structural studies by electron tomography: from cells to molecules. Annu Rev Biochem 74:833–865" href="/article/10.1007/s10055-009-0129-y#ref-CR2" id="ref-link-section-d20606e329">2005</a>) and small-angle X-ray scattering (SAXS; Petoukhov and Svergun <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2007" title="Petoukhov MV, Svergun DI (2007) Analysis of X-ray and neutron scattering from biomacromolecular solutions. Curr Opin Struct Biol 17:562–571" href="/article/10.1007/s10055-009-0129-y#ref-CR3" id="ref-link-section-d20606e332">2007</a>), also yield non-atomic resolution 3D reconstructions of protein complexes and the methods discussed in this paper are equally applicable.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1"><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig1_HTML.jpg?as=webp"></source><img aria-describedby="figure-1-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig1_HTML.jpg" alt="figure1" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>The interface of the Sculptor visualization package, showing a 3D volumetric reconstruction (from cryo-EM) of a GroEL chaperonin</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/1" data-track-dest="link:Figure1 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                     <p>The failure to achieve atomic resolution with cryo-EM, cryo-ET, or SAXS is not as big a stumbling block as one might expect. Often, crystal structures or good homology models are available for individual subunits. These known atomic structures can be docked into experimental volumetric reconstructions, yielding an atomic model of the whole complex (Steven and Baumeister <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2008" title="Steven AC, Baumeister W (2008) The future is hybrid. J Struct Biol 163:186–195" href="/article/10.1007/s10055-009-0129-y#ref-CR4" id="ref-link-section-d20606e356">2008</a>; Wriggers and Chacón <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2001" title="Wriggers W, Chacón P (2001) Modeling tricks and fitting techniques for multiresolution structures. Structure 9:779–788" href="/article/10.1007/s10055-009-0129-y#ref-CR5" id="ref-link-section-d20606e359">2001</a>; Rossmann et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2005" title="Rossmann MG, Morais MC, Leiman PG, Zhang W (2005) Combining X-ray crystallography and electron microscopy. Structure 13:355–362" href="/article/10.1007/s10055-009-0129-y#ref-CR6" id="ref-link-section-d20606e362">2005</a>). Currently, various docking approaches exist, the results of which have lead to a significantly better understanding of the structure and interaction of proteins and protein complexes. However, the multi-resolution docking problem poses a series of challenges, which are detailed in Sect. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/article/10.1007/s10055-009-0129-y#Sec2">2</a>. Those challenges cannot be dealt with by fully automated, exhaustive search-based, pattern matching algorithms. User input is vital and an immersive environment is key to allow the user to efficiently dock high-resolution structures into experimental volumetric data sets. The goal of our present work is to provide a fully immersive docking environment, which utilizes stereoscopic viewing, head tracking, and haptic rendering. Only such a truly interactive approach allows an in-depth, exhaustive, and efficient exploration of a given docking problem.</p><p>The present report first discusses both docking challenges and existing fitting methods, and then describes our novel technique.</p></div></div></section><section aria-labelledby="Sec2"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Docking challenges and existing methods</h2><div class="c-article-section__content" id="Sec2-content"><p>The multi-resolution docking problem can be defined in the present context as the following: given a 3D volumetric map and a set of atomic structures, find the locations and orientations of the structures that best reproduce the volume data. Theoretically, maximizing a scoring function that measures the quality of the current docking will yield the desired result. Each structure should be fitted independently to avoid problems if not all atomic structures are known.</p><p>While the ultimate goal, stated above, seems feasible, numerous problems appear due to the nature of the experimental data sets. The actual micrographs obtained by cryo-EM contain large amounts of noise (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig2">2</a>), caused by the extremely low electron dose necessary to avoid destroying the specimen. The low signal-to-noise ratio can be improved by first aligning and then averaging particle projections of identical orientations. In practice, tens of thousands of individual particles images are required to produce a 3D reconstruction of 10 Å resolution or better (Frank <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2006" title="Frank J (2006) Three-dimensional electron microscopy of macromolecular assemblies. Oxford University Press, New York" href="/article/10.1007/s10055-009-0129-y#ref-CR7" id="ref-link-section-d20606e383">2006</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig2_HTML.jpg?as=webp"></source><img aria-describedby="figure-2-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig2_HTML.jpg" alt="figure2" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>Six cryo-electron micrographs of the GroEL chaperonin</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/2" data-track-dest="link:Figure2 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                     <p>Several other factors also reduce the resolution or introduce distortions. First, no two protein complexes are truly identical due to small random conformational changes. Second, multiple stable conformations may exist, which differ significantly but are difficult to distinguish in the micrographs. Third, the density of the volume data is often heterogeneous, again caused by conformational fluctuations. Fourth, the orientational preferences of a biomolecule on the experimental support can lead to major gaps in the angular space of the projections and cause deformations in the 3D reconstruction. Finally, due to the complex nature of cryo-EM, artifacts may be introduced at any stage from the sample preparation to recording the actual micrograph.</p><p>The experimental difficulties of obtaining a good 3D reconstruction of a protein complex represent only one class of problems. Further challenges arise from the atomic structures used for the docking. X-ray structures may exhibit distortions due to crystal packing effects. Homology models are usually derived from different species and are only approximate models of the true atomic structure of the proteins under study.</p><p>All the above factors contribute in rendering multi-resolution docking an ill-posed problem. While the underlying theoretical problem is straightforward, including these complicating factors into a scoring function is difficult and may never be achieved. Therefore, the knowledge and reasoning of a human user will always be necessary to critically evaluate solutions to a given docking problem.</p><p>Two general approaches for solving the multi-resolution docking problem are currently prevalent: exhaustive evaluation of cross-correlation-based scoring functions (Chacón and Wriggers <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2002" title="Chacón P, Wriggers W (2002) Multi-resolution contour-based fitting of macromolecular structures. J Mol Biol 317:375–384" href="/article/10.1007/s10055-009-0129-y#ref-CR8" id="ref-link-section-d20606e414">2002</a>; Jiang et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2001" title="Jiang W, Baker ML, Ludtke SJ, Chiu W (2001) Bridging the information gap: computational tools for intermediate resolution structure interpretation. J Mol Biol 308:1033–1044" href="/article/10.1007/s10055-009-0129-y#ref-CR9" id="ref-link-section-d20606e417">2001</a>; Garzón et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2007" title="Garzón JI, Kovacs JA, Abagyan R, Chacón P (2007) ADP_EM: fast exhaustive multi-resolution docking for high-throughput coverage. Bioinformatics 23:427–433" href="/article/10.1007/s10055-009-0129-y#ref-CR10" id="ref-link-section-d20606e420">2007</a>; Volkmann and Hanein <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1999" title="Volkmann N, Hanein D (1999) Quantitative fitting of atomic models into observed densities by electron microscopy. J Struct Biol 125:176–184" href="/article/10.1007/s10055-009-0129-y#ref-CR11" id="ref-link-section-d20606e423">1999</a>; Roseman <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2000" title="Roseman AM (2000) Docking structures of domains into maps from cryo-electron microscopy using local correlation. Acta Crystallogr D 56:1332–1340" href="/article/10.1007/s10055-009-0129-y#ref-CR12" id="ref-link-section-d20606e426">2000</a>; Rossmann <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2000" title="Rossmann MG (2000) Fitting atomic models into electron-microscopy maps. Acta Crystallogr D 56:1341–1349" href="/article/10.1007/s10055-009-0129-y#ref-CR13" id="ref-link-section-d20606e430">2000</a>; Ceulemans and Russell <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2004" title="Ceulemans H, Russell RB (2004) Fast fitting of atomic structures to low-resolution electron density maps by surface overlap maximization. J Mol Biol 338:783–793" href="/article/10.1007/s10055-009-0129-y#ref-CR14" id="ref-link-section-d20606e433">2004</a>; Wu et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2003" title="Wu X, Milne JLS, Borgnia MJ, Rostapshov AV, Subramaniam S, Brooks BR (2003) A core-weighted fitting method for docking atomic structures into low-resolution maps: application to cryo-electron microscopy. J Struct Biol 141:63–76" href="/article/10.1007/s10055-009-0129-y#ref-CR15" id="ref-link-section-d20606e436">2003</a>) or measuring the deviation of feature points of the multi-resolution 3D objects (Wriggers et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1999" title="Wriggers W, Milligan RA, McCammon JA (1999) Situs: a package for docking crystal structures into low-resolution maps from electron microscopy. J Struct Biol 125:185–195" href="/article/10.1007/s10055-009-0129-y#ref-CR16" id="ref-link-section-d20606e439">1999</a>; Birmanns and Wriggers <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2007" title="Birmanns S, Wriggers W (2007) Multi-resolution anchor-point registration of biomolecular assemblies and their components. J Struct Biol 157:271–280" href="/article/10.1007/s10055-009-0129-y#ref-CR17" id="ref-link-section-d20606e442">2007</a>). Each approach has different strengths and weaknesses—the method proposed in this work utilizes a cross-correlation coefficient as its main scoring function, an adaptation to feature-based scoring functions would also be possible.</p><p>In correlation-based docking, the scoring function is determined by projecting and blurring the atomic structure onto a 3D map and calculating the cross-correlation coefficient between this map and the original volume data. The statistical nature of cross-correlation ensures that high scores are generated not only for nearly perfect matches, but also for approximate agreement. This property is vital in the current context since near-perfect matches are rare. Using cross-correlation as a scoring function, the best docking locations are usually found via an exhaustive search. Such a docking procedure can be applied successfully to high-quality, intermediate resolution maps, but is prone to fail for more problematic data sets. In the past, efforts went into the development of advanced scoring functions that aim to counteract the artifacts of the experimental maps. An example is the introduction of an edge-detection filter like the Laplacian (Chacón and Wriggers <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2002" title="Chacón P, Wriggers W (2002) Multi-resolution contour-based fitting of macromolecular structures. J Mol Biol 317:375–384" href="/article/10.1007/s10055-009-0129-y#ref-CR8" id="ref-link-section-d20606e448">2002</a>), which extends significantly the viable resolution range of automatic molecular-docking tools. Nevertheless, as stated above, the resolution is just one property of the volumetric map—numerous factors contribute to the final shape of the volume data and it is not feasible to model them accurately using a scoring function.</p><p>Besides the scoring function itself, exhaustive search methods encounter a second problem: How are candidate solutions picked? Solely relying on high docking scores is only practical in near-perfect data sets. In less optimal cases, many almost equivalent solutions exist and the user is inundated by hundreds of solutions for a protein complex containing only a few subunits.</p><p>To illustrate this point further, we chose the GroEL chaperonin (PDB entry 1GRL) as a test system. This protein is a homo-14mer with a monomer weight of approximately 47 kDa. We first performed a series of exhaustive searches at several resolutions. The atomic structure of 1GRL was low pass filtered, using a Gaussian kernel, to 10, 12.5, and 15 Å  resolution. To simulate experimental density variations, the weight of atoms in the outer “rings” was varied between 1.00 and 0.70. This yielded a density decrease in these parts similar to that found in experimental maps (where it is due to motion of atoms in these regions). Using the simulated maps, 23 exhaustive searches were performed. For an automated search to succeed, the top 14 solution candidates have to correspond to the 14 monomers in the protein complex. As Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/article/10.1007/s10055-009-0129-y#Tab1">1</a> shows, the exhaustive search can only be trusted at 10 Å  resolution, with little or no density variation. These are the sole cases where the top 14 candidates represent the 14 monomers in the structure. At lower resolutions or larger density variations, the exhaustive search fails to produce a proper ranking of solutions. While all true solutions are still found, they are preceded by large numbers of false positives (see the “highest correct” column in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/article/10.1007/s10055-009-0129-y#Tab1">1</a>).</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 Summary of simulated GroEL docking results</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/article/10.1007/s10055-009-0129-y/tables/1"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                     <p>As an alternative to the fully automatic docking tools, scientists often carry out purely user-guided docking in a visualization program (Kleywegt et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2001" title="Kleywegt GJ, Zou JY, Kjeldgaard M, Jones TA (2001) Around O. International tables for crystallography, vol F, chap 17.1. Kluwer, Dordrecht, pp 353–356, 366–367" href="/article/10.1007/s10055-009-0129-y#ref-CR18" id="ref-link-section-d20606e975">2001</a>). This solely visual approach allows a biologist to directly apply his or her knowledge of the system under study. On the other hand, the procedure is highly subjective and the software does not support the user in any way. Manipulating a protein in six dimensions (6D, three translations and three rotations) is non-trivial and the best docking solutions are not necessarily evident.</p><p>In related scientific fields, virtual reality has been used successfully to enhance scientists’ understanding of a given problem and guide software towards promising solutions. Tactile feedback for drug-receptor docking simulations has been pioneered by Ouh-young et al. (<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1988" title="Ouh-young M, Pique M, Hughes J, Srinivasan N, Brooks FP Jr (1988) Using a manipulator for force display in molecular docking. In: 1988 IEEE robotics and automation conference, vol 3. Philadelphia, PA, pp 1824–1829" href="/article/10.1007/s10055-009-0129-y#ref-CR19" id="ref-link-section-d20606e982">1988</a>). More recently, Bayazit et al. (<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2001" title="Bayazit OB, Song G, Amato NM (2001) Ligand binding with OBPRM and haptic user input. In: 2001 IEEE international conference on robotics and automation. Seoul, Korea" href="/article/10.1007/s10055-009-0129-y#ref-CR20" id="ref-link-section-d20606e985">2001</a>) also used force feedback to guide a motion planning algorithm for ligand binding. In the present context of multi-resolution docking, our group has developed a haptic docking solution which computes a simplified scoring function and the related forces on the fly (Birmanns and Wriggers <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2003" title="Birmanns S, Wriggers W (2003) Interactive fitting augmented by force-feedback and virtual reality. J Struct Biol 144:123–131" href="/article/10.1007/s10055-009-0129-y#ref-CR21" id="ref-link-section-d20606e988">2003</a>).</p><p>Our current aim is to support the user with significantly more docking information and provide a fully immersive environment for solving challenging multi-resolution docking problems. Tactile clues about a scoring landscape are critical in such an endeavor, but they need to be based on an accurate scoring function. High-quality scoring functions are computationally expensive and therefore not suitable for real-time updates. We circumvent this problem by utilizing a fast exhaustive search pattern matching platform which can pre-compute a 3D field of fitting scores. Such a field is amicable to visualization, haptic rendering, and further feature extraction. Since the scoring field is calculated offline, complex, correlation based, scoring functions can be used and the interactive portion of our system can focus on generating additional docking information on the fly. One crucial piece of information is the interactions with already docked proteins. We developed an intuitive system which provides visual cues to the user about favorable interactions and highlights potential clashes. This combination of offline exhaustive search and online interaction detection allows an intuitive, visual, and haptic exploration of a given docking problem.</p></div></div></section><section aria-labelledby="Sec3"><div class="c-article-section" id="Sec3-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec3">Interactive global docking</h2><div class="c-article-section__content" id="Sec3-content"><p>Interactive global docking (IGD) consists of two separate steps: first, an offline exhaustive search is performed in Eliquos, our new cross-correlation based exhaustive search software. This typically takes from a few minutes to a few hours, depending on the size of the system. The exhaustive search produces a vector field containing fitting scores and orientations of the probe structure, i.e., a scoring field. This field is read into our Sculptor visualization package and the user interactively explores both the scoring field and any additional docking information generated on-the-fly by Sculptor. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig3">3</a> shows an outline of the software architecture and the sections below discuss both steps in detail.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3"><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig3_HTML.gif?as=webp"></source><img aria-describedby="figure-3-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig3_HTML.gif" alt="figure3" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>The overall software architecture of the interactive global docking system</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/3" data-track-dest="link:Figure3 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                     <h3 class="c-article__sub-heading" id="Sec4">Exhaustive search</h3><p>Eliquos, which performs the offline exhaustive search, is a general multi-resolution docking application. It is geared towards fast and accurate docking of large data sets, using a scoring function based on cross-correlation. Several advanced filtering techniques are available to enable high-accuracy docking at both high and low resolutions. Like most exhaustive search docking programs, Eliquos is a black box in the sense that once the user specifies the input parameters, no further interaction occurs and the software will output a set of solutions guided solely by the chosen scoring function.</p><p>Eliquos is written in C, employing extensive application-specific low-level tuning, OpenMP, and the Intel Math Kernel Library (Intel Corporation <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1994–2008" title="Intel Corporation (1994–2008) Intel math kernel library 10.0" href="/article/10.1007/s10055-009-0129-y#ref-CR22" id="ref-link-section-d20606e1031">1994–2008</a>) to achieve high performance. It uses the standard MRC file format (Short <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2006" title="Short J (2006) MRC Cambridge Image Processing System 28 Sep 2006" href="/article/10.1007/s10055-009-0129-y#ref-CR23" id="ref-link-section-d20606e1034">2006</a>) for volume data I/O and the PDB format (The worldwide Protein Data Bank <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2008" title="The worldwide Protein Data Bank (2008) Protein data bank contents guide: atomic coordinate entry format description" href="/article/10.1007/s10055-009-0129-y#ref-CR24" id="ref-link-section-d20606e1037">2008</a>) for atomic structures. While the actual docking procedure follows the approach of Chacón and Wriggers (<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2002" title="Chacón P, Wriggers W (2002) Multi-resolution contour-based fitting of macromolecular structures. J Mol Biol 317:375–384" href="/article/10.1007/s10055-009-0129-y#ref-CR8" id="ref-link-section-d20606e1040">2002</a>), Eliquos contains many improvements to increase the efficiency of the code. The two most prominent new features are pre-screening of possible orientations and a padding routine which extends the experimental map by the smallest amount necessary for Fast Fourier Transform (FFT) translational matching.</p><p>The six degrees of freedom (DOF) which are examined in a 6D exhaustive search are treated in two different ways. The translational DOF are explored via the application of the Fourier convolution theorem, which states that a convolution in real space is equivalent to a multiplication in Fourier space. Calculating the cross-correlation (CC) in Fourier space allows the simultaneous evaluation of the CC for all possible docking locations in a given map (using the voxels of the target map as an implicit grid). The three rotational DOF are scanned explicitly using a uniform angular grid. Typically, an angular grid resolution of under 10° is needed for sufficiently fine sampling, which leads to a large number of samples. At 9° angular step size, over 20,000 Euler angles need to be explored. Since the execution time of the docking algorithm scales linearly with the number of angles examined, it is judicious to prune the angular grid as much as possible. Eliquos achieves this pruning by testing each orientation and determining if the orientation is feasible from a purely geometric point of view. For example, the experimental map of a membrane protein will be much larger in one direction than in the other two, limiting the viable orientations of the equally elongated probe structure. On the other hand, when a small probe structure is docked into a very large protein complex, most (if not all) orientations need to be examined. Eliquos performs this screening procedure quickly by calculating the convex hull (Barber et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1996" title="Barber CB, Dobkin DP, Huhdanpaa HT (1996) The quickhull algorithm for convex hulls. ACM Trans Math Software 22:469–483" href="/article/10.1007/s10055-009-0129-y#ref-CR25" id="ref-link-section-d20606e1046">1996</a>) of the probe structure. The convex hull describes an envelope around the probe and only contains the (small number of) atoms which span this envelope. The convex hull can, therefore, be used to rapidly determine which orientations will fit inside the experimental map.</p><p>Having found all feasible orientations, Eliquos then determines the amount of padding needed for the experimental volumetric map. Padding the map is necessary for several reasons: a given orientation might be viable but cannot fit into the map when rotated around the center of the probe structure, padding needed for filtering the map, padding to avoid FFT artifacts during the translational search, and padding to enable the use of the most efficient FFT pathways available. The actual amount of padding is determined taking all four factors into account. This approach ensures that only the absolute minimum of padding is added since the cost of the FFT scales as <span class="mathjax-tex">\({\mathcal{O}}(n \log n)\)</span>, where <i>n</i> is the total number of voxels.</p><p>In a final preparation step, a list of reasonable probe positions is generated. Again, purely geometric reasoning is applied to determine all possible positions of the probe structure. This step is necessary since Laplacian filtering, e.g., can generate false positive solutions, corresponding to exterior docking. In practice, Eliquos generates a spherical test volume with a diameter of half the smallest extent of the probe structure. A single unfiltered FFT cross-correlation calculation is then performed, which results in a 3D map of viable locations.</p><p>After all preparation steps are completed, the exhaustive 6D search can be performed (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig4">4</a>). The search loops over all feasible orientations, rotating and blurring the probe structure, and calculating the cross-correlation coefficients for all possible translations in the given orientation. At the end of each iteration, a list of candidate solutions is updated to reflect the currently best orientation at each position.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4"><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig4_HTML.gif?as=webp"></source><img aria-describedby="figure-4-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig4_HTML.gif" alt="figure4" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>The main loop of the 6D exhaustive search in Eliquos</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/4" data-track-dest="link:Figure4 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                        <p>If a fully automated exhaustive search is desired, Eliquos then locates the highest ranked positions and corresponding orientations. The set of candidate solutions is sorted by docking score and then pruned to eliminate near-degenerate solutions. The resulting solutions are further optimized using a local optimization procedure. This off-grid optimization ensures the best possible docking of the high-resolution probe structure into the experimental 3D reconstruction.</p><p>In the case of IGD, Eliquos instead generates several files containing the scoring field. These files consist of two MRC volume data sets and one text file with a list of all sampled orientations. The first volume file contains the scalar component of the scoring field, i.e., the score of the best solution at each position. The second volume holds the corresponding orientation in the form of an index. The Sculptor visualization package can read in and use this scoring field as one of the docking criteria supporting the user’s fitting decisions.</p><h3 class="c-article__sub-heading" id="Sec5">Interactive visualization</h3><p>The interactive stage of IGD is performed in the Sculptor visualization package. Sculptor (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig1">1</a>) is aimed primarily at working with volumetric data sets and docking of high-resolution structures into these 3D reconstructions. It makes extensive use of hardware graphics acceleration to provide high performance visualization of large data sets. Some of the provided features are as follows: volume manipulation tools, isosurface and direct volume rendering, feature-based multi-resolution docking using vector quantization, fast flexible fitting based on interpolation, and cross-correlation-based refinement of approximately fitted structures. In addition, Sculptor provides broad virtual reality support: stereoscopic rendering, head tracking, and various haptic devices are supported. The paragraphs below first describe the general software architecture of Sculptor and then outline the main loop of the IGD visualization phase. Finally, both the haptic rendering and steric interaction components are explained in more detail.</p><p>The application Sculptor is a thin software layer sitting on top of a general purpose scientific visualization and virtual reality library called SVT. SVT was developed for various in-house virtual reality projects, providing applications with a transparent software platform to access immersive, multi-display VR systems, and also conventional workstations. The C++ library combines a flexible VR rendering backend with advanced scientific visualization methods. Beside volume rendering and biomolecular visualization routines, the toolkit also includes support for tactile feedback using a variety of different haptic devices. The modules controlling the virtual reality and haptic rendering hardware are designed as a separate abstraction layer called LIVE. The LIVE layer provides a device-independent interface and permits flexible, on-demand loading of code. This way, devices can be added or removed after linking the main application, and even during run-time. In addition, the library makes VR devices accessible through the network and across different operating system platforms, without any changes in the application code.</p><p>Sculptor itself links SVT with biomolecular modeling routines and a Qt-based user interface. It thereby can run not only on most multi-display VR installations, but also on PC workstations typically found in experimental biology labs. If executed on a workstation, the user interface integrates the 3D rendering in the main application window (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig1">1</a>), to assure a similar user experience as in other 3D modeling packages. In a VR system, the 3D rendering area is detached from the user interface and shown on the display areas using a perspective correction based on the position information from the head-tracking device.</p><p>For IGD, Sculptor first loads the exhaustive search results generated by Eliquos. When IGD is activated, the main loop shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig5">5</a> is entered. As the user moves the probe structure, Sculptor retrieves both the optimal orientation at the current position and the global docking score from the exhaustive search data. Then, the forces are calculated from the scoring field via quadratic interpolation. Next, steric interactions are determined and finally both the visual and haptic output is updated. Once a satisfactory probe position is found, it can be added to a set of candidate solutions. The user can easily switch the focus of IGD between the current position and any of the saved solutions, allowing an iterative refinement of the docking locations.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5"><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig5_HTML.gif?as=webp"></source><img aria-describedby="figure-5-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig5_HTML.gif" alt="figure5" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>Sculptor: main loop during interactive global docking</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/5" data-track-dest="link:Figure5 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                        <p>Sculptor has already supported force-feedback guided docking for some time (Birmanns and Wriggers <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2003" title="Birmanns S, Wriggers W (2003) Interactive fitting augmented by force-feedback and virtual reality. J Struct Biol 144:123–131" href="/article/10.1007/s10055-009-0129-y#ref-CR21" id="ref-link-section-d20606e1144">2003</a>). A local cross-correlation measure was used to give feedback about the local neighborhood and guide the user to favorable docking positions. This approach has the distinct advantage that no pre-computation is necessary and the user can start exploring the docking problem immediately. However, a computationally simple docking score must be used in this context, since haptic devices need real-time force updates in the order of 1,000 Hz to provide a smooth user experience. Full cross-correlation scores with advanced filtering are computationally too expensive. By using a pre-computed scoring field, scoring functions of arbitrary complexity can be used and the force calculation simplifies to a table lookup and trivial interpolation. The forces are determined via the first derivative of a quadratic polynomial, using the current score and its nearest neighbors as input data points for the interpolant. The interpolation and differentiation are performed separately along each principal axis. The orientations contained in the scoring field allow the software to rotate the probe structure into the best orientation at the current point in space. The user is thus only responsible for translating the structure, which greatly simplifies his task. Limiting the interactive search space to 3D also has the advantage that inexpensive haptic devices can be used for the force feedback. Novint Technologies, Inc. (<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2008" title="Novint Technologies Inc. (2008) The Falcon haptic device. &#xA;                    http://www.novint.com&#xA;                    &#xA;                  &#xA;                        " href="/article/10.1007/s10055-009-0129-y#ref-CR26" id="ref-link-section-d20606e1147">2008</a>), e.g., offers the Falcon 3D device priced below $200. While the Falcon was developed for immersive 3D gaming, it is ideally suited for our docking approach. The device only supports 3D translations but no rotations. In practice, the user experience with the Falcon is at least as good as, e.g., with a SensAble Phantom. Since only 3D translations are needed for IGD, the lack of rotational DOF is actually an advantage.</p><p>During the interactive stage of IGD, Sculptor not only uses the scoring field generated by Eliquos, but also generates additional, on-the-fly, docking information which the user can draw on to find ideal docking positions. The additional information is based on possible steric interactions between candidate solutions (i.e., favorable protein–protein distances). The aim is not to fully simulate complex protein–protein interactions but rather highlight areas of interest for the user. Whenever a new potential solution is found by the user, Sculptor calculates a 3D distance map which contains the distance from any point in space to the nearest backbone atom of all solutions. These distances are stored as a 3D volume where each voxel contains the corresponding distance. An example of such a map is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig6">6</a>. The current candidate solutions are shown in light blue. The red and green volumes highlight forbidden and favorable positions for backbone atoms of the probe structure, respectively. This distance map allows a very efficient determination of good and bad protein interactions. The map is consulted at the positions of all <i>C</i>
                           <sub>α</sub> atoms of the probe structure and the distance is compared to known interaction ranges.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6"><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig6_HTML.jpg?as=webp"></source><img aria-describedby="figure-6-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig6_HTML.jpg" alt="figure6" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p>Sculptor screenshot: example of a distance map used for efficient steric interaction determination. The current candidate solutions are shown in <i>light blue</i>. If any backbone atoms of the probe structure enter a <i>red region</i>, a steric clash with the existing solutions is likely. On the other hand, the <i>green regions</i> represent areas resulting in favorable interaction distances.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/6" data-track-dest="link:Figure6 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                        <p>Källblad and Dean (<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2004" title="Källblad P, Dean PM (2004) Backbonebackbone geometry of tertiary contacts between α-helices. Proteins 56:693–703" href="/article/10.1007/s10055-009-0129-y#ref-CR27" id="ref-link-section-d20606e1192">2004</a>) performed an extensive study of protein–protein contacts in nature. They concluded that only a narrow range of backbone–backbone distances occur at the interface between two proteins. Based on their results, we consider distances below 2.5 Å to represent a clash and 2.5–10 Å to be a favorable interaction. While these default values appear to be suitable for a wide range of docking problems, they can nonetheless be changed interactively.</p></div></div></section><section aria-labelledby="Sec6"><div class="c-article-section" id="Sec6-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec6">Computational and VR details</h2><div class="c-article-section__content" id="Sec6-content"><p>Our VR setup is a self-built, low-cost back-projection system, employing polarization filters and inexpensive polarized glasses for stereoscopic viewing. Two DLP projectors with linear polarizing filters and a polarization preserving screen are used. Due to space constraints, two mirrors fold the light path, yielding a compact system with a 80 × 80 footprint and a 80 × 60 screen. An electro-magnetic head-tracking system is used to determine the viewer’s position and viewing direction. The projected view is continuously updated to provide the illusion of stationary 3D objects.</p><p>The display system is driven by an AMD Athlon64 4400+ X2 dual core processor with 2 GB of RAM and a NVidia Quadro FX 4500 graphics card. A SensAble Phantom 1.5/6DOF, connected via a parallel port, provides 6D input and force feedback. To garantee high haptic rendering update frequencies, the device control and force calculations are performed in a separate thread. Due to the pre-computed scoring field, this thread only consumes a fraction of one CPU core.</p><p>An inexpensive haptic device alternative for the desktop is the Novint Falcon. While not providing quite the fidelity of the Phantom, the Falcon is perfectly suitable for desktops or even laptops. Paired with visual force output via an arrow, it makes IGD also accessible on common and inexpensive hardware.</p><p>Sculptor, Eliquos, and the associated libraries are freely available in binary form from <a href="http://sculptor.biomachina.org">http://sculptor.biomachina.org</a>. Executables for 32 and 64 bit Linux, Windows, Power and Intel Macs are available. Please contact the authors for access to the source code.</p><p>We also ported the exhaustive search export functionality into the established Colores tool (Chacón and Wriggers <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2002" title="Chacón P, Wriggers W (2002) Multi-resolution contour-based fitting of macromolecular structures. J Mol Biol 317:375–384" href="/article/10.1007/s10055-009-0129-y#ref-CR8" id="ref-link-section-d20606e1221">2002</a>) of the Situs docking package (available from <a href="http://situs.biomachina.org">http://situs.biomachina.org</a>). This allows users to take advantage of IGD without requiring knowledge of the new Eliquos package.</p></div></div></section><section aria-labelledby="Sec7"><div class="c-article-section" id="Sec7-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">IGD in practice</h2><div class="c-article-section__content" id="Sec7-content"><p>To give an example of docking with IGD, we return to the GroEL example given earlier. We start by loading the scoring field from the 15 Å  resolution test with a density variation of 0.9 into Sculptor. The top scoring solution of the exhaustive search is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>a and is clearly incorrect. As Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/article/10.1007/s10055-009-0129-y#Tab1">1</a> shows, the first <i>correct</i> solution has a rank of 295, but the scoring field does contain <i>all</i> correct solutions. The IGD starting position (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>b) again reflects this high scoring but false solution. The probe structure can then be moved in 3D via a haptic device (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>c). The user is only responsible for translating the center of the monomer, the optimal orientation at the present position is determined by the scoring field. So, as the probe structure is moved around the experimental data set, it automatically rotates into the most favorable orientation. In addition, the (globally normalized) score at the present position is displayed graphically through color changes of the central sphere as well as numerically. The user then locates a suitable candidate position for the first docked monomer, taking the global docking score and his or her knowledge of the system into account. When trying to find such a position, which should be a local maximum in the scoring field, the force feedback becomes a crucial guide for the user and is further enhanced by visual feedback via a gradient arrow. When a local maximum is encountered, it is highlighted by a wire-frame overlay around the central sphere (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>d). Once a suitable location is chosen, it can be saved as a solution (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>e, white structure) and the probe is moved in search for the next candidate location. At this stage, the additional steric information generated by Sculptor comes into play. The user can visualize steric clashes between the current probe structure and all previously docked solutions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>e) as well as good protein contacts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig7">7</a>f). Once all constituent proteins are approximately placed, their positions can be further adjusted by either manual or automatic refinement. For example, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig8">8</a> shows a closer view of the fit resulting from automatic Laplacian refinement of all subunits. No steric clashes are evident and the protein interfaces are highlighted by green spheres, signifying good contacts.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7"><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 7</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig7_HTML.jpg?as=webp"></source><img aria-describedby="figure-7-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig7_HTML.jpg" alt="figure7" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p>Stages of interactive global docking. The figure shows direct screen captures from the Sculptor visualization package. <b>a</b> Incorrect top scoring solution from exhaustive search. <b>b</b> Starting point of IGD. <b>c</b> Moving the probe. <b>d</b> Found a local maximum. <b>e</b> Steric clashes shown as <i>red spheres</i>. <b>f</b> Good contacts highlighted as <i>green spheres</i>
                                 </p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/7" data-track-dest="link:Figure7 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                        <div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-8"><figure><figcaption><b id="Fig8" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 8</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/8" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig8_HTML.jpg?as=webp"></source><img aria-describedby="figure-8-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig8_HTML.jpg" alt="figure8" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p>Results of automated refinement of the interactively docked structures. No steric clashes occur and the protein–protein interfaces are highlighted by <i>green spheres</i>
                                 </p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/8" data-track-dest="link:Figure8 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                     </div></div></section><section aria-labelledby="Sec8"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Interactive docking example: binding of Cdc6 to the origin recognition complex (ORC)</h2><div class="c-article-section__content" id="Sec8-content"><p>Binding of the Cdc6 protein to the ORC is a crucial step in the assembly of the DNA pre-replication complex (Liang et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1995" title="Liang C, Weinreich M, Stillman B (1995) ORC and Cdc6p interact and determine the frequency of initiation of DNA replication in the genome. Cell 81:667–676" href="/article/10.1007/s10055-009-0129-y#ref-CR28" id="ref-link-section-d20606e1349">1995</a>) during cell division. The ORC–Cdc6 assembly has previously been extensively studied by Speck et al. (<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2005" title="Speck C, Chen Z, Li H, Stillman B (2005) ATPase-dependent cooperative binding of ORC and Cdc6 to origin DNA. Nat Struct Mol Biol 12:965–971" href="/article/10.1007/s10055-009-0129-y#ref-CR29" id="ref-link-section-d20606e1352">2005</a>) using various techniques, including negative stain electron microscopy. The resulting volumetric data sets with 20 Å resolution are publicly available (EMDB entries 1156 and 1157). Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig9">9</a>a shows a map of the ORC by itself and Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig9">9</a>b depicts the assembled ORC–Cdc6 complex. While a significant reorganization occurs in the ORC during assembly, Cdc6 clearly docks on the left side of the complex.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-9"><figure><figcaption><b id="Fig9" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 9</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/9" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig9_HTML.gif?as=webp"></source><img aria-describedby="figure-9-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs10055-009-0129-y/MediaObjects/10055_2009_129_Fig9_HTML.gif" alt="figure9" loading="lazy" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-9-desc"><p>Interactive docking of ORC–Cdc6: <b>a</b> Experimental map of ORC without Cdc6. <b>b</b> Experimental map of ORC with Cdc6. <b>c</b> Globally best docking solution from exhaustive search. <b>d</b> Docking solution from IGD with subsequent local optimization.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/article/10.1007/s10055-009-0129-y/figures/9" data-track-dest="link:Figure9 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                     <p>Using PDB entry 1FNN (Liu et al. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2000" title="Liu J, Smith CL, DeRyckere D, DeAngelis K, Martin GS, Berger JM (2000) Structure and function of Cdc6/Cdc18: implications for origin recognition and checkpoint control. Mol Cell 6:637–648" href="/article/10.1007/s10055-009-0129-y#ref-CR30" id="ref-link-section-d20606e1395">2000</a>) as the Cdc6 probe structure, we performed an exhaustive search with Laplacian filtering and refined the ten top scoring candidate solutions. An angular step size of 9° was used during the search, resulting in a search space of 20,400 angles. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig9">9</a>c shows the ORC–Cdc6 complex with the top scoring docked solution (blue). However, this solution lies completely within the original ORC and can, therefore, not be valid. In fact, all the top ten solutions fall within the ORC part of the complex. These false positives are produced by the comparatively low resolution of 20 Å, a lack of interior details in negative stain maps, and an uneven density distribution in the experimental data.</p><p>Exploring the exhaustive search data via IGD in Sculptor yielded five candidate solutions in the correct area of the ORC–Cdc6 complex. A local optimization (again using Laplacian filtering) was performed on these solutions. The optimized candidates were ranked both via standard and Laplacian cross-correlation. Two of the candidates produce nearly identical top scores; however, one of them again showed significant overlap with the lone ORC structure. The remaining solution is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/article/10.1007/s10055-009-0129-y#Fig9">9</a>d and is similar to the manually docked structure by Speck et al.</p><p>The present example highlights the true power of IGD. Often, information about the rough position of a subunit is known from biology, but traditional exhaustive search methods fail to rank solutions in this area high enough. Thus, they are often not seen by the user. As mentioned above, uneven density distributions in the experimental data usually prevent automatic exhaustive search algorithms from succeeding in such cases. Picking the correct candidate solution then falls back on the user’s knowledge of the system’s biology and IGD greatly simplifies this process.</p></div></div></section><section aria-labelledby="Sec9"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Conclusions</h2><div class="c-article-section__content" id="Sec9-content"><p>The IGD approach, presented here, combines the best features of non-interactive exhaustive search techniques and purely interactive visualization methods. It provides the user with both visual and haptic feedback about global docking scores, steric clashes, and good protein–protein interfaces. The additional information supplied during IGD allows biologists to not only rely on their personal knowledge of the system but also draw on objective, software-generated, fitting information. The currently available indicators represent only the first steps for incorporating more information into multi-resolution docking procedures. In the future, we plan to include contact information from mutation experiments, distances from NMR or FRET quenching measurements, as well as improving the existing scoring functions. Finally, the ORC–Cdc6 binding example demonstrates how our new approach allows for an efficient solution of a docking problem where traditional exhaustive search techniques fail. The highly immersive environment, provided by IGD, greatly facilitates this docking task for the user.</p></div></div></section>
                        
                    

                    <section aria-labelledby="Bib1"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="CB. Barber, DP. Dobkin, HT. Huhdanpaa, " /><meta itemprop="datePublished" content="1996" /><meta itemprop="headline" content="Barber CB, Dobkin DP, Huhdanpaa HT (1996) The quickhull algorithm for convex hulls. ACM Trans Math Software 22" /><p class="c-article-references__text" id="ref-CR25">Barber CB, Dobkin DP, Huhdanpaa HT (1996) The quickhull algorithm for convex hulls. ACM Trans Math Software 22:469–483</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1145%2F235815.235821" aria-label="View reference 1">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ams.org/mathscinet-getitem?mr=1428265" aria-label="View reference 1 on MathSciNet">MathSciNet</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.emis.de/MATH-item?0884.65145" aria-label="View reference 1 on MATH">MATH</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 1 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20quickhull%20algorithm%20for%20convex%20hulls&amp;journal=ACM%20Trans%20Math%20Software&amp;volume=22&amp;pages=469-483&amp;publication_year=1996&amp;author=Barber%2CCB&amp;author=Dobkin%2CDP&amp;author=Huhdanpaa%2CHT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Bayazit OB, Song G, Amato NM (2001) Ligand binding with OBPRM and haptic user input. In: 2001 IEEE internation" /><p class="c-article-references__text" id="ref-CR20">Bayazit OB, Song G, Amato NM (2001) Ligand binding with OBPRM and haptic user input. In: 2001 IEEE international conference on robotics and automation. Seoul, Korea</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Birmanns, W. Wriggers, " /><meta itemprop="datePublished" content="2003" /><meta itemprop="headline" content="Birmanns S, Wriggers W (2003) Interactive fitting augmented by force-feedback and virtual reality. J Struct Bi" /><p class="c-article-references__text" id="ref-CR21">Birmanns S, Wriggers W (2003) Interactive fitting augmented by force-feedback and virtual reality. J Struct Biol 144:123–131</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jsb.2003.09.018" aria-label="View reference 3">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 3 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Interactive%20fitting%20augmented%20by%20force-feedback%20and%20virtual%20reality&amp;journal=J%20Struct%20Biol&amp;volume=144&amp;pages=123-131&amp;publication_year=2003&amp;author=Birmanns%2CS&amp;author=Wriggers%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Birmanns, W. Wriggers, " /><meta itemprop="datePublished" content="2007" /><meta itemprop="headline" content="Birmanns S, Wriggers W (2007) Multi-resolution anchor-point registration of biomolecular assemblies and their " /><p class="c-article-references__text" id="ref-CR17">Birmanns S, Wriggers W (2007) Multi-resolution anchor-point registration of biomolecular assemblies and their components. J Struct Biol 157:271–280</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jsb.2006.08.008" aria-label="View reference 4">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 4 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-resolution%20anchor-point%20registration%20of%20biomolecular%20assemblies%20and%20their%20components&amp;journal=J%20Struct%20Biol&amp;volume=157&amp;pages=271-280&amp;publication_year=2007&amp;author=Birmanns%2CS&amp;author=Wriggers%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Ceulemans, RB. Russell, " /><meta itemprop="datePublished" content="2004" /><meta itemprop="headline" content="Ceulemans H, Russell RB (2004) Fast fitting of atomic structures to low-resolution electron density maps by su" /><p class="c-article-references__text" id="ref-CR14">Ceulemans H, Russell RB (2004) Fast fitting of atomic structures to low-resolution electron density maps by surface overlap maximization. J Mol Biol 338:783–793</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jmb.2004.02.066" aria-label="View reference 5">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 5 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20fitting%20of%20atomic%20structures%20to%20low-resolution%20electron%20density%20maps%20by%20surface%20overlap%20maximization&amp;journal=J%20Mol%20Biol&amp;volume=338&amp;pages=783-793&amp;publication_year=2004&amp;author=Ceulemans%2CH&amp;author=Russell%2CRB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="P. Chacón, W. Wriggers, " /><meta itemprop="datePublished" content="2002" /><meta itemprop="headline" content="Chacón P, Wriggers W (2002) Multi-resolution contour-based fitting of macromolecular structures. J Mol Biol 31" /><p class="c-article-references__text" id="ref-CR8">Chacón P, Wriggers W (2002) Multi-resolution contour-based fitting of macromolecular structures. J Mol Biol 317:375–384</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1006%2Fjmbi.2002.5438" aria-label="View reference 6">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 6 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-resolution%20contour-based%20fitting%20of%20macromolecular%20structures&amp;journal=J%20Mol%20Biol&amp;volume=317&amp;pages=375-384&amp;publication_year=2002&amp;author=Chac%C3%B3n%2CP&amp;author=Wriggers%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Frank, " /><meta itemprop="datePublished" content="2002" /><meta itemprop="headline" content="Frank J (2002) Single-particle imaging of macromolecules by cryo-electron microscopy. Annu Rev Biophys Biomol " /><p class="c-article-references__text" id="ref-CR1">Frank J (2002) Single-particle imaging of macromolecules by cryo-electron microscopy. Annu Rev Biophys Biomol Struct 31:303–319</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1146%2Fannurev.biophys.31.082901.134202" aria-label="View reference 7">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 7 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-particle%20imaging%20of%20macromolecules%20by%20cryo-electron%20microscopy&amp;journal=Annu%20Rev%20Biophys%20Biomol%20Struct&amp;volume=31&amp;pages=303-319&amp;publication_year=2002&amp;author=Frank%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><meta itemprop="author" content="J. Frank, " /><meta itemprop="datePublished" content="2006" /><meta itemprop="headline" content="Frank J (2006) Three-dimensional electron microscopy of macromolecular assemblies. Oxford University Press, Ne" /><p class="c-article-references__text" id="ref-CR7">Frank J (2006) Three-dimensional electron microscopy of macromolecular assemblies. Oxford University Press, New York</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 8 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Three-dimensional%20electron%20microscopy%20of%20macromolecular%20assemblies&amp;publication_year=2006&amp;author=Frank%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JI. Garzón, JA. Kovacs, R. Abagyan, P. Chacón, " /><meta itemprop="datePublished" content="2007" /><meta itemprop="headline" content="Garzón JI, Kovacs JA, Abagyan R, Chacón P (2007) ADP_EM: fast exhaustive multi-resolution docking for high-thr" /><p class="c-article-references__text" id="ref-CR10">Garzón JI, Kovacs JA, Abagyan R, Chacón P (2007) ADP_EM: fast exhaustive multi-resolution docking for high-throughput coverage. Bioinformatics 23:427–433</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl625" aria-label="View reference 9">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 9 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ADP_EM%3A%20fast%20exhaustive%20multi-resolution%20docking%20for%20high-throughput%20coverage&amp;journal=Bioinformatics&amp;volume=23&amp;pages=427-433&amp;publication_year=2007&amp;author=Garz%C3%B3n%2CJI&amp;author=Kovacs%2CJA&amp;author=Abagyan%2CR&amp;author=Chac%C3%B3n%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Intel Corporation (1994–2008) Intel math kernel library 10.0" /><p class="c-article-references__text" id="ref-CR22">Intel Corporation (1994–2008) Intel math kernel library 10.0</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="W. Jiang, ML. Baker, SJ. Ludtke, W. Chiu, " /><meta itemprop="datePublished" content="2001" /><meta itemprop="headline" content="Jiang W, Baker ML, Ludtke SJ, Chiu W (2001) Bridging the information gap: computational tools for intermediate" /><p class="c-article-references__text" id="ref-CR9">Jiang W, Baker ML, Ludtke SJ, Chiu W (2001) Bridging the information gap: computational tools for intermediate resolution structure interpretation. J Mol Biol 308:1033–1044</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1006%2Fjmbi.2001.4633" aria-label="View reference 11">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 11 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Bridging%20the%20information%20gap%3A%20computational%20tools%20for%20intermediate%20resolution%20structure%20interpretation&amp;journal=J%20Mol%20Biol&amp;volume=308&amp;pages=1033-1044&amp;publication_year=2001&amp;author=Jiang%2CW&amp;author=Baker%2CML&amp;author=Ludtke%2CSJ&amp;author=Chiu%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="P. Källblad, PM. Dean, " /><meta itemprop="datePublished" content="2004" /><meta itemprop="headline" content="Källblad P, Dean PM (2004) Backbonebackbone geometry of tertiary contacts between α-helices. Proteins 56:693–7" /><p class="c-article-references__text" id="ref-CR27">Källblad P, Dean PM (2004) Backbonebackbone geometry of tertiary contacts between α-helices. Proteins 56:693–703</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1002%2Fprot.20201" aria-label="View reference 12">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 12 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Backbonebackbone%20geometry%20of%20tertiary%20contacts%20between%20%CE%B1-helices&amp;journal=Proteins&amp;volume=56&amp;pages=693-703&amp;publication_year=2004&amp;author=K%C3%A4llblad%2CP&amp;author=Dean%2CPM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Kleywegt GJ, Zou JY, Kjeldgaard M, Jones TA (2001) Around O. International tables for crystallography, vol F, " /><p class="c-article-references__text" id="ref-CR18">Kleywegt GJ, Zou JY, Kjeldgaard M, Jones TA (2001) Around O. International tables for crystallography, vol F, chap 17.1. Kluwer, Dordrecht, pp 353–356, 366–367</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Liang, M. Weinreich, B. Stillman, " /><meta itemprop="datePublished" content="1995" /><meta itemprop="headline" content="Liang C, Weinreich M, Stillman B (1995) ORC and Cdc6p interact and determine the frequency of initiation of DN" /><p class="c-article-references__text" id="ref-CR28">Liang C, Weinreich M, Stillman B (1995) ORC and Cdc6p interact and determine the frequency of initiation of DNA replication in the genome. Cell 81:667–676</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2F0092-8674%2895%2990528-6" aria-label="View reference 14">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 14 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ORC%20and%20Cdc6p%20interact%20and%20determine%20the%20frequency%20of%20initiation%20of%20DNA%20replication%20in%20the%20genome&amp;journal=Cell&amp;volume=81&amp;pages=667-676&amp;publication_year=1995&amp;author=Liang%2CC&amp;author=Weinreich%2CM&amp;author=Stillman%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Liu, CL. Smith, D. DeRyckere, K. DeAngelis, GS. Martin, JM. Berger, " /><meta itemprop="datePublished" content="2000" /><meta itemprop="headline" content="Liu J, Smith CL, DeRyckere D, DeAngelis K, Martin GS, Berger JM (2000) Structure and function of Cdc6/Cdc18: i" /><p class="c-article-references__text" id="ref-CR30">Liu J, Smith CL, DeRyckere D, DeAngelis K, Martin GS, Berger JM (2000) Structure and function of Cdc6/Cdc18: implications for origin recognition and checkpoint control. Mol Cell 6:637–648</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS1097-2765%2800%2900062-9" aria-label="View reference 15">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 15 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20and%20function%20of%20Cdc6%2FCdc18%3A%20implications%20for%20origin%20recognition%20and%20checkpoint%20control&amp;journal=Mol%20Cell&amp;volume=6&amp;pages=637-648&amp;publication_year=2000&amp;author=Liu%2CJ&amp;author=Smith%2CCL&amp;author=DeRyckere%2CD&amp;author=DeAngelis%2CK&amp;author=Martin%2CGS&amp;author=Berger%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="V. Lučić, F. Förster, W. Baumeister, " /><meta itemprop="datePublished" content="2005" /><meta itemprop="headline" content="Lučić V, Förster F, Baumeister W (2005) Structural studies by electron tomography: from cells to molecules. An" /><p class="c-article-references__text" id="ref-CR2">Lučić V, Förster F, Baumeister W (2005) Structural studies by electron tomography: from cells to molecules. Annu Rev Biochem 74:833–865</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1146%2Fannurev.biochem.73.011303.074112" aria-label="View reference 16">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 16 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20studies%20by%20electron%20tomography%3A%20from%20cells%20to%20molecules&amp;journal=Annu%20Rev%20Biochem&amp;volume=74&amp;pages=833-865&amp;publication_year=2005&amp;author=Lu%C4%8Di%C4%87%2CV&amp;author=F%C3%B6rster%2CF&amp;author=Baumeister%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Novint Technologies Inc. (2008) The Falcon haptic device. http://www.novint.com&#xA;                        " /><p class="c-article-references__text" id="ref-CR26">Novint Technologies Inc. (2008) The Falcon haptic device. <a href="http://www.novint.com">http://www.novint.com</a>
                        </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Ouh-young M, Pique M, Hughes J, Srinivasan N, Brooks FP Jr (1988) Using a manipulator for force display in mol" /><p class="c-article-references__text" id="ref-CR19">Ouh-young M, Pique M, Hughes J, Srinivasan N, Brooks FP Jr (1988) Using a manipulator for force display in molecular docking. In: 1988 IEEE robotics and automation conference, vol 3. Philadelphia, PA, pp 1824–1829</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MV. Petoukhov, DI. Svergun, " /><meta itemprop="datePublished" content="2007" /><meta itemprop="headline" content="Petoukhov MV, Svergun DI (2007) Analysis of X-ray and neutron scattering from biomacromolecular solutions. Cur" /><p class="c-article-references__text" id="ref-CR3">Petoukhov MV, Svergun DI (2007) Analysis of X-ray and neutron scattering from biomacromolecular solutions. Curr Opin Struct Biol 17:562–571</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.sbi.2007.06.009" aria-label="View reference 19">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 19 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20X-ray%20and%20neutron%20scattering%20from%20biomacromolecular%20solutions&amp;journal=Curr%20Opin%20Struct%20Biol&amp;volume=17&amp;pages=562-571&amp;publication_year=2007&amp;author=Petoukhov%2CMV&amp;author=Svergun%2CDI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="AM. Roseman, " /><meta itemprop="datePublished" content="2000" /><meta itemprop="headline" content="Roseman AM (2000) Docking structures of domains into maps from cryo-electron microscopy using local correlatio" /><p class="c-article-references__text" id="ref-CR12">Roseman AM (2000) Docking structures of domains into maps from cryo-electron microscopy using local correlation. Acta Crystallogr D 56:1332–1340</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444900010908" aria-label="View reference 20">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 20 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Docking%20structures%20of%20domains%20into%20maps%20from%20cryo-electron%20microscopy%20using%20local%20correlation&amp;journal=Acta%20Crystallogr%20D&amp;volume=56&amp;pages=1332-1340&amp;publication_year=2000&amp;author=Roseman%2CAM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MG. Rossmann, " /><meta itemprop="datePublished" content="2000" /><meta itemprop="headline" content="Rossmann MG (2000) Fitting atomic models into electron-microscopy maps. Acta Crystallogr D 56:1341–1349" /><p class="c-article-references__text" id="ref-CR13">Rossmann MG (2000) Fitting atomic models into electron-microscopy maps. Acta Crystallogr D 56:1341–1349</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444900009562" aria-label="View reference 21">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 21 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Fitting%20atomic%20models%20into%20electron-microscopy%20maps&amp;journal=Acta%20Crystallogr%20D&amp;volume=56&amp;pages=1341-1349&amp;publication_year=2000&amp;author=Rossmann%2CMG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MG. Rossmann, MC. Morais, PG. Leiman, W. Zhang, " /><meta itemprop="datePublished" content="2005" /><meta itemprop="headline" content="Rossmann MG, Morais MC, Leiman PG, Zhang W (2005) Combining X-ray crystallography and electron microscopy. Str" /><p class="c-article-references__text" id="ref-CR6">Rossmann MG, Morais MC, Leiman PG, Zhang W (2005) Combining X-ray crystallography and electron microscopy. Structure 13:355–362</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.str.2005.01.005" aria-label="View reference 22">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 22 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Combining%20X-ray%20crystallography%20and%20electron%20microscopy&amp;journal=Structure&amp;volume=13&amp;pages=355-362&amp;publication_year=2005&amp;author=Rossmann%2CMG&amp;author=Morais%2CMC&amp;author=Leiman%2CPG&amp;author=Zhang%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Short J (2006) MRC Cambridge Image Processing System 28 Sep 2006" /><p class="c-article-references__text" id="ref-CR23">Short J (2006) MRC Cambridge Image Processing System 28 Sep 2006</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Speck, Z. Chen, H. Li, B. Stillman, " /><meta itemprop="datePublished" content="2005" /><meta itemprop="headline" content="Speck C, Chen Z, Li H, Stillman B (2005) ATPase-dependent cooperative binding of ORC and Cdc6 to origin DNA. N" /><p class="c-article-references__text" id="ref-CR29">Speck C, Chen Z, Li H, Stillman B (2005) ATPase-dependent cooperative binding of ORC and Cdc6 to origin DNA. Nat Struct Mol Biol 12:965–971</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 24 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ATPase-dependent%20cooperative%20binding%20of%20ORC%20and%20Cdc6%20to%20origin%20DNA&amp;journal=Nat%20Struct%20Mol%20Biol&amp;volume=12&amp;pages=965-971&amp;publication_year=2005&amp;author=Speck%2CC&amp;author=Chen%2CZ&amp;author=Li%2CH&amp;author=Stillman%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="AC. Steven, W. Baumeister, " /><meta itemprop="datePublished" content="2008" /><meta itemprop="headline" content="Steven AC, Baumeister W (2008) The future is hybrid. J Struct Biol 163:186–195" /><p class="c-article-references__text" id="ref-CR4">Steven AC, Baumeister W (2008) The future is hybrid. J Struct Biol 163:186–195</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jsb.2008.06.002" aria-label="View reference 25">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 25 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20future%20is%20hybrid&amp;journal=J%20Struct%20Biol&amp;volume=163&amp;pages=186-195&amp;publication_year=2008&amp;author=Steven%2CAC&amp;author=Baumeister%2CW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="The worldwide Protein Data Bank (2008) Protein data bank contents guide: atomic coordinate entry format descri" /><p class="c-article-references__text" id="ref-CR24">The worldwide Protein Data Bank (2008) Protein data bank contents guide: atomic coordinate entry format description</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="N. Volkmann, D. Hanein, " /><meta itemprop="datePublished" content="1999" /><meta itemprop="headline" content="Volkmann N, Hanein D (1999) Quantitative fitting of atomic models into observed densities by electron microsco" /><p class="c-article-references__text" id="ref-CR11">Volkmann N, Hanein D (1999) Quantitative fitting of atomic models into observed densities by electron microscopy. J Struct Biol 125:176–184</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1006%2Fjsbi.1998.4074" aria-label="View reference 27">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 27 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantitative%20fitting%20of%20atomic%20models%20into%20observed%20densities%20by%20electron%20microscopy&amp;journal=J%20Struct%20Biol&amp;volume=125&amp;pages=176-184&amp;publication_year=1999&amp;author=Volkmann%2CN&amp;author=Hanein%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="W. Wriggers, P. Chacón, " /><meta itemprop="datePublished" content="2001" /><meta itemprop="headline" content="Wriggers W, Chacón P (2001) Modeling tricks and fitting techniques for multiresolution structures. Structure 9" /><p class="c-article-references__text" id="ref-CR5">Wriggers W, Chacón P (2001) Modeling tricks and fitting techniques for multiresolution structures. Structure 9:779–788</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0969-2126%2801%2900648-7" aria-label="View reference 28">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 28 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20tricks%20and%20fitting%20techniques%20for%20multiresolution%20structures&amp;journal=Structure&amp;volume=9&amp;pages=779-788&amp;publication_year=2001&amp;author=Wriggers%2CW&amp;author=Chac%C3%B3n%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="W. Wriggers, RA. Milligan, JA. McCammon, " /><meta itemprop="datePublished" content="1999" /><meta itemprop="headline" content="Wriggers W, Milligan RA, McCammon JA (1999) Situs: a package for docking crystal structures into low-resolutio" /><p class="c-article-references__text" id="ref-CR16">Wriggers W, Milligan RA, McCammon JA (1999) Situs: a package for docking crystal structures into low-resolution maps from electron microscopy. J Struct Biol 125:185–195</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1006%2Fjsbi.1998.4080" aria-label="View reference 29">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 29 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Situs%3A%20a%20package%20for%20docking%20crystal%20structures%20into%20low-resolution%20maps%20from%20electron%20microscopy&amp;journal=J%20Struct%20Biol&amp;volume=125&amp;pages=185-195&amp;publication_year=1999&amp;author=Wriggers%2CW&amp;author=Milligan%2CRA&amp;author=McCammon%2CJA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Wu, JLS. Milne, MJ. Borgnia, AV. Rostapshov, S. Subramaniam, BR. Brooks, " /><meta itemprop="datePublished" content="2003" /><meta itemprop="headline" content="Wu X, Milne JLS, Borgnia MJ, Rostapshov AV, Subramaniam S, Brooks BR (2003) A core-weighted fitting method for" /><p class="c-article-references__text" id="ref-CR15">Wu X, Milne JLS, Borgnia MJ, Rostapshov AV, Subramaniam S, Brooks BR (2003) A core-weighted fitting method for docking atomic structures into low-resolution maps: application to cryo-electron microscopy. J Struct Biol 141:63–76</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS1047-8477%2802%2900570-1" aria-label="View reference 30">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 30 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20core-weighted%20fitting%20method%20for%20docking%20atomic%20structures%20into%20low-resolution%20maps%3A%20application%20to%20cryo-electron%20microscopy&amp;journal=J%20Struct%20Biol&amp;volume=141&amp;pages=63-76&amp;publication_year=2003&amp;author=Wu%2CX&amp;author=Milne%2CJLS&amp;author=Borgnia%2CMJ&amp;author=Rostapshov%2CAV&amp;author=Subramaniam%2CS&amp;author=Brooks%2CBR">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="/article/10.1007/s10055-009-0129-y-references.ris">Download references<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section><section aria-labelledby="Ack1"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgments</h2><div class="c-article-section__content" id="Ack1-content"><p>We would like to thank Huilin Li for kindly discussing the ORC–Cdc6 results with us and Willy Wriggers for his valuable advice regarding this project. The present work was supported by a training fellowship (to J.H.) from the National Library of Medicine to the Keck Center for Interdisciplinary Bioscience Training of the Gulf Coast Consortia (NLM Grant No. 5T15LM- 07093), NIH grant R01GM62968, a grant from the Gillson-Longenbaugh Foundation, and startup funds from the University of Texas at Houston (to S.B.). In addition, we would like to thank the reviewers for many helpful comments and suggestions.</p>
                <h3 class="c-article__sub-heading">Open Access</h3>
                <p>This article is distributed under the terms of the Creative Commons Attribution Noncommercial License which permits any noncommercial use, distribution, and reproduction in any medium, provided the original author(s) and source are credited.</p>
              </div></div></section><section aria-labelledby="author-information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">School of Health Information Sciences, University of Texas at Houston, 7000 Fannin St., Houston, TX, 77030, USA</p><p class="c-article-author-affiliation__authors-list">Jochen Heyd &amp; Stefan Birmanns</p></li></ol><div class="js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Jochen-Heyd"><span class="c-article-authors-search__title u-h3 js-search-name">Jochen Heyd</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?dc.creator=&#34;Jochen+Heyd&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jochen+Heyd" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jochen+Heyd%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Stefan-Birmanns"><span class="c-article-authors-search__title u-h3 js-search-name">Stefan Birmanns</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?dc.creator=&#34;Stefan+Birmanns&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stefan+Birmanns" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stefan+Birmanns%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" rel="nofollow" href="/article/10.1007/s10055-009-0129-y/email/correspondent/c1/new">Stefan Birmanns</a>.</p></div></div></section><section aria-labelledby="rightslink"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b> This is an open access article distributed under the terms of the Creative Commons Attribution Noncommercial License (<a href="https://creativecommons.org/licenses/by-nc/2.0" rel="license" itemprop="license">https://creativecommons.org/licenses/by-nc/2.0</a>), which permits any noncommercial use, distribution, and reproduction in any medium, provided the original author(s) and source are credited.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Immersive%20structural%20biology%3A%20a%20new%20approach%20to%20hybrid%20modeling%20of%20macromolecular%20assemblies&amp;author=Jochen%20Heyd%20et%20al&amp;contentID=10.1007%2Fs10055-009-0129-y&amp;publication=1359-4338&amp;publicationDate=2009-08-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY-NC">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Heyd, J., Birmanns, S. Immersive structural biology: a new approach to hybrid modeling of macromolecular assemblies.
                    <i>Virtual Reality</i> <b>13, </b>245–255 (2009). https://doi.org/10.1007/s10055-009-0129-y</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" href="/article/10.1007/s10055-009-0129-y.ris">Download citation<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2009-02-28">28 February 2009</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2009-07-14">14 July 2009</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2009-08-26">26 August 2009</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2009-12">December 2009</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--doi"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value"><a href="https://doi.org/10.1007/s10055-009-0129-y" data-track="click" data-track-action="view doi" data-track-label="link" itemprop="sameAs">https://doi.org/10.1007/s10055-009-0129-y</a></span></p></li></ul><div data-component="share-box"></div><h3 class="c-article__sub-heading">Keywords</h3><ul class="c-article-subject-list"><li class="c-article-subject-list__subject"><span itemprop="about">Structural biology</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Docking</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Multi-resolution modeling</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Electron microscopy</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Haptic rendering</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Virtual reality</span></li></ul><div data-component="article-info-list"></div></div></div></div></div></section>
                </div>
            </article>
        </main>

        <div class="c-article-extras u-text-sm u-hide-print" id="sidebar" data-container-type="reading-companion" data-track-component="reading companion">
            <aside>
                <div data-test="download-article-link-wrapper">
                    
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://link.springer.com/content/pdf/10.1007/s10055-009-0129-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="button">
            
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            
        </a>
    </div>
    

                </div>

                <div data-test="collections">
                    
                </div>

                <div data-test="editorial-summary">
                    
                </div>

                <div class="c-reading-companion">
                    <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
                        

                        <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                            <div class="js-ad">
    <aside class="c-ad c-ad--300x250">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-MPU1" data-gpt-unitpath="/270604982/springerlink/10055/article" data-gpt-sizes="300x250" data-gpt-targeting="pos=MPU1;articleid=129;"></div>
        </div>
    </aside>
</div>

                        </div>
                        <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
                        <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
                    </div>
                </div>
            </aside>
        </div>
    </div>


        
    <footer class="app-footer" role="contentinfo">
        <div class="app-footer__aside-wrapper u-hide-print">
            <div class="app-footer__container">
                <p class="app-footer__strapline">Over 10 million scientific documents at your fingertips</p>
                
                    <div class="app-footer__edition" data-component="SV.EditionSwitcher">
                        <span class="u-visually-hidden" data-role="button-dropdown__title" data-btn-text="Switch between Academic & Corporate Edition">Switch Edition</span>
                        <ul class="app-footer-edition-list" data-role="button-dropdown__content" data-test="footer-edition-switcher-list">
                            <li class="selected">
                                <a data-test="footer-academic-link"
                                   href="/siteEdition/link"
                                   id="siteedition-academic-link">Academic Edition</a>
                            </li>
                            <li>
                                <a data-test="footer-corporate-link"
                                   href="/siteEdition/rd"
                                   id="siteedition-corporate-link">Corporate Edition</a>
                            </li>
                        </ul>
                    </div>
                
            </div>
        </div>
        <div class="app-footer__container">
            <ul class="app-footer__nav u-hide-print">
                <li><a href="/">Home</a></li>
                <li><a href="/impressum">Impressum</a></li>
                <li><a href="/termsandconditions">Legal information</a></li>
                <li><a href="/privacystatement">Privacy statement</a></li>
                <li><a href="https://www.springernature.com/ccpa">California Privacy Statement</a></li>
                <li><a href="/cookiepolicy">How we use cookies</a></li>
                
                <li><a class="optanon-toggle-display" href="javascript:void(0);">Manage cookies/Do not sell my data</a></li>
                
                <li><a href="/accessibility">Accessibility</a></li>
                <li><a id="contactus-footer-link" href="/contactus">Contact us</a></li>
            </ul>
            <div class="c-user-metadata">
    
        <p class="c-user-metadata__item">
            <span data-test="footer-user-login-status">Not logged in</span>
            <span data-test="footer-user-ip"> - 130.225.98.201</span>
        </p>
        <p class="c-user-metadata__item" data-test="footer-business-partners">
            Copenhagen University Library Royal Danish Library Copenhagen (1600006921)  - DEFF Consortium Danish National Library Authority (2000421697)  - Det Kongelige Bibliotek The Royal Library (3000092219)  - DEFF Danish Agency for Culture (3000209025)  - 1236 DEFF LNCS (3000236495) 
        </p>

        
    
</div>

            <a class="app-footer__parent-logo" target="_blank" rel="noopener" href="//www.springernature.com"  title="Go to Springer Nature">
                <span class="u-visually-hidden">Springer Nature</span>
                <svg width="125" height="12" focusable="false" aria-hidden="true">
                    <image width="125" height="12" alt="Springer Nature logo"
                           src=/oscar-static/images/springerlink/png/springernature-60a72a849b.png
                           xmlns:xlink="http://www.w3.org/1999/xlink"
                           xlink:href=/oscar-static/images/springerlink/svg/springernature-ecf01c77dd.svg>
                    </image>
                </svg>
            </a>
            <p class="app-footer__copyright">&copy; 2020 Springer Nature Switzerland AG. Part of <a target="_blank" rel="noopener" href="//www.springernature.com">Springer Nature</a>.</p>
            
        </div>
        
    <svg class="u-hide hide">
        <symbol id="global-icon-chevron-right" viewBox="0 0 16 16">
            <path d="M7.782 7L5.3 4.518c-.393-.392-.4-1.022-.02-1.403a1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L7.782 9l1.013-.998z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-download" viewBox="0 0 16 16">
            <path d="M2 14c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H3.002A1.006 1.006 0 012 14zM9 2v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L3.115 7.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L7 8.8V2c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-email" viewBox="0 0 18 18">
            <path d="M1.995 2h14.01A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM1 13.994A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3H1.995A1 1 0 001 4.006zM9 11L2 7V5.557l7 4 7-4V7z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-institution" viewBox="0 0 18 18">
            <path d="M14 8a1 1 0 011 1v6h1.5a.5.5 0 01.5.5v.5h.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H1v-.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v6h8V9a1 1 0 011-1zM6 8l2 1v4l-2 1zm6 0v6l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-search" viewBox="0 0 22 22">
            <path fill-rule="evenodd" d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z"/>
        </symbol>
    </svg>

    </footer>



    </div>
    
    
</body>
</html>

